mikeio1d/result_reader_writer/result_reader.py

```diff
@@ -45,6 +56,16 @@ class ResultReader:
         Flag specifying to add chainage into data frame column label.
     """
 
+    class ColumnMode(str, Enum):
```

Comment:
I suggest to take out the enum one level up. I think the nested classes/enums like that are fine if they are really private helper classes.